reading/arithmetic.livemd

```diff
@@ -67,7 +67,7 @@ Often you'll use integers for representing ages, days, years, cash values, and m
 
 ### Your Turn
 
-In the Elixir cell below, replace `0` the biggest integer you can think of (until you get bored).
+In the Elixir cell below, replace `1` the biggest integer you can think of (until you get bored).
```

Comment:
This change is outside the scope of the original issue. Not sure how strict PRs need to be. Happy to extract this if wanted.

Reply:
No problem!